src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_StrandBiasTest.java

```diff
@@ -0,0 +1,229 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import org.apache.log4j.Logger;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.genotyper.MostLikelyAllele;
+import org.broadinstitute.hellbender.utils.genotyper.PerReadAlleleLikelihoodMap;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
+
+import java.util.*;
+
+/**
+ * Allele-specific implementation of strand bias annotations
+ */
+public abstract class AS_StrandBiasTest extends StrandBiasTest implements ReducibleAnnotation {
+    private final static Logger logger = Logger.getLogger(AS_StrandBiasTest.class);
+    protected static final String splitDelim = "\\|"; //String.split takes a regex, so we need to escape the pipe
+    protected static final String printDelim = "|";
```

Comment:
Same delimiter expressions are repeated in several classes -- should they be declared as a shared constant somewhere?